webapp/src/main/java/com/box/l10n/mojito/service/branch/notification/slack/BranchNotificationMessageBuilderSlack.java

```diff
@@ -132,4 +139,15 @@ String getSummaryString(List<String> strings) {
         .map(s -> StringUtils.abbreviate(s, STRING_IN_SUMMARY_ABRREVIATE_LENGHT))
         .collect(Collectors.joining(", "));
   }
+
+  protected String getTranslationsReadyMsg(String branchName) {
+    final ImmutableMap.Builder<String, Object> messageParamMap = ImmutableMap.builder();
+    messageParamMap.put("branchName", branchName);
+    if (isGithubPR) {
+      GithubBranchDetails branchDetails = new GithubBranchDetails(branchName);
+      messageParamMap.put("githubRepository", branchDetails.getRepository());
```

Comment:
In my original comment, I suggested `github.repository`. This uses `githubRepository`. Is this due to implementation limitation with the former or a stylistic choice? 

@garionpin any preference on the format to use?

Reply:
Yes, I have answered that above.
![image](https://github.com/box/mojito/assets/44348924/8542c81d-b4a2-4131-ad3e-3864138b1d05)